app/static/src/css/custom.css

```diff
@@ -26,3 +26,13 @@
 .success-text {
     color: #ffffff;
 }
+
+.govuk-header__content {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.govuk-header__navigation {
+  margin-left: auto;
+}
```

Comment:
Do we have to do this via custom css? Have you checked to see if there are existing classes for this styling?

Reply:
Yes, I have checked the existing classes and am confident that this is the best approach, the way it was orignially done was quite "hacky" imp because the service name was entered as a navigation item

Reply:
Sounds good!